cloudconfig.tpl

```diff
@@ -725,6 +725,8 @@ write_files:
     permissions: 0775
     path: /home/wowza/wowza-mount.sh
     content: |
+        echo wowza-mount.sh Last Run Date $(date) > /home/wowza/wowza-mount-last-run.txt
```

Comment:
Can we add this to the 'azLogs' folder so it is synced to Azure Storage please?

Reply:
Updated